test/testapps/BasicTestApp/InteropTest/TreeNode.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@using Microsoft.AspNetCore.Blazor.Components
```

Comment:
Doesn't look like this component is used anywhere. Or am I missing something?

Reply:
Yeah, this needs cleanup.